### DISCUSSION TITLE
does the plugin mqtt-proxy support SSL?

### DISCUSSION THREAD

**Discussion by liandimx** [apache/apisix#6106] [Q&A]
document says "The plugin mqtt-proxy only works in stream model, it help you to dynamic load balance by client_id of MQTT."
interesting but i wonder if the plugin supports MQTT messages over SSL?
if it does, do we need any configuration for it?
if it does not, is there any plan to support SSL in the future?
thank you.

**Comment by juzhiyuan** [comment]
Hi @shuaijinchao, could you please have a confirmation?

**Comment by believe1499** [comment]
Is there any suggestion for this requirement？

**Reply by tokers** [reply]
Try to create an SSL object, see https://apisix.apache.org/docs/apisix/next/admin-api/ for details.
